Added docs for setting up executorch on windows

nil-is-all opened the pull request:
### Summary
First documentation for setting up ExecuTorch for Windows and exporting MobileNet V2 model and executing.
This version works only on Windows Powershell on admin mode.

nil-is-all committed:
Added docs for setting up executorch on windows

nil-is-all commented:
@pytorchbot label "release notes: none"

nil-is-all commented:
lintrunner --take NEWLINE -a

nil-is-all committed:
removed trailing newline at bottom

nil-is-all commented:
@GregoryComer @SS-JIA Feel free to tag anyone else as well to help review this draft on Windows Doc

mergennachin commented:
Why is this a completely different page?

Can we do cards where you can toggle between different codeblocks based on platform

See https://docs.pytorch.org/torchrec/setup-torchrec.html#installation

cc @GregoryComer

SS-JIA reviewed (approved):
Awesome stuff!

mergennachin reviewed (changes requested):
See my previous comment 

cc @SS-JIA

GregoryComer commented on docs/source/using-executorch-windows.md:
````diff
@@ -0,0 +1,149 @@
+## Run Mobilenet V2 model with XNNPACK delegation
+
+```bash
+.\cmake-out\backends\xnnpack\Release\xnn_executor_runner.exe --model_path=.\mv2_xnnpack.pte
````
Can you escape the backslashes (replace \ with \\, I believe) here? It renders weirdly in the docs due to interpreting as a escape char - see https://docs-preview.pytorch.org/pytorch/executorch/11273/using-executorch-windows.html.

GregoryComer commented:
> Why is this a completely different page?
> 
> Can we do cards where you can toggle between different codeblocks based on platform
> 
> See https://docs.pytorch.org/torchrec/setup-torchrec.html#installation as an example where you can toggle based (pypi, nightly, build from source)
> 
> cc @GregoryComer

@nil-is-all I'll defer to Mergen's preference on where to put this. Can you take a look at trying to integrate into the existing [Building from Source](https://docs.pytorch.org/executorch/stable/using-executorch-building-from-source.html) page? The code is in using-executorch-building-from-source.md.

We can probably just add a Windows section initially, and we can look at maybe integrating [sphinx-tabs](https://sphinx-tabs.readthedocs.io/en/latest/) or something eventually. There's enough Windows specific steps that I'm fine with having a dedicated windows heading for now.

GregoryComer commented on docs/source/using-executorch-windows.md:
```diff
@@ -0,0 +1,149 @@
+This document outlines the current known working build instructions for building and validating ExecuTorch on a Windows machine.
+
+This demo uses the
+[MobileNet v2](https://pytorch.org/vision/main/models/mobilenetv2.html) model to
```
I'd maybe change the wording here slightly since the current runner doesn't have camera integration. Maybe replace with "uses the MobileNet v2 model to classify images.".

GregoryComer commented on docs/source/using-executorch-windows.md:
```diff
@@ -0,0 +1,149 @@
+
+
+## Create the Conda Environment
+To check if conda is detected by the powershell prompt, try `conda list` or `conda --version`
```
Nit: Maybe reword as "To verify that Conda is available in the in the powershell environment, run  try `conda list` or `conda --version`. If Conda is not available, run `conda-hook.ps1` as follows:

GregoryComer commented on docs/source/using-executorch-windows.md:
````diff
@@ -0,0 +1,149 @@
+```bash
+Output 0: tensor(sizes=[1, 1000], [
+  -0.50986, 0.30064, 0.0953904, 0.147726, 0.231205, 0.338555, 0.206892, -0.0575775, … ])
+```
````
Can we maybe add a short comment explaining that these values represent class scores?

nil-is-all committed:
Edits based on new suggestions

1) Changed MV2 intro to "uses the MobileNet v2 model to classify images." since current runner doesn't have camera integration
2) reworded conda env setup
3) added escape character for backslashes
4) mentioned MV2 model output values represent class scores

nil-is-all commented:
@GregoryComer I reflected the smaller edits you mentioned. Will work on integrating into the Building from Source page as a section.

nil-is-all committed:
Added Windows docs

nil-is-all committed:
Delete docs/source/using-executorch-windows.md

Added as a section in using-executorch-building-from-source.md
